src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -122,23 +122,23 @@ public String getRank() {
 	}
 	
 	public void setRank(String rank) {
-		this.rank = rank;
+		this.rank = Utils.trimStringReturnNull(rank);
 	}
 	
 	public String getBiography() {
 		return biography;
 	}
 	
 	public void setBiography(String biography) {
-		this.biography = biography;
+		this.biography = Utils.trimStringReturnNull(biography);
 	}
 
 	public String getDomainUsername() {
 		return domainUsername;
 	}
 
 	public void setDomainUsername(String domainUsername) {
-		this.domainUsername = domainUsername;
+		this.domainUsername = Utils.trimStringReturnNull(domainUsername);
 	}
```

Comment:
I don't think domainUsername should be trimmed, as this one comes from AD ... I think.